docs-ref-conceptual/python-sdk-azure-install.md

```diff
@@ -69,4 +69,23 @@ If you want to install `azure` from source:
 
     git clone git://github.com/Azure/azure-sdk-for-python.git
     cd azure-sdk-for-python
-    python setup.py install
\ No newline at end of file
+    python setup.py install
+
+## Install an older version with pip
+You can install an older version of `azure` by specifying 'azure==3.0.0' verion details.
+```bash
+pip install azure==3.0.0 
+```
+## Check SDK Installation details with pip
```

Comment:
Sentence-style casing for all headings: Check SDK installation details with pip

Reply:
updated